### PR TITLE
lots of minor edits

### PR DESCRIPTION
**Pull request by pkra**

### CODE REVIEW

**Comment by pkra** [comment]
I'm afraid hard wraps mess up GitHub's preview. I'll add some inline comments.

File: sloan-draft.tex [hunk 1]
```diff
@@ -15,52 +15,45 @@
...
-response to the report \emph{Developing a 21st Century Global Library
-  for Mathematics Research} written under the auspices of the National
-Research Council, and given further form as a new Global Digital
-Mathematics Library initiative.\comment{VS}{What's the relationship
-  between Sloan and this?} The project will aim to bridge the gap
-between the current state of retro-digitised mathematical literature,
-where only textual content is recognised while mathematical formulas
-are omitted, and the need to make the mathematical material fully
-accessible to researchers and computational software
-(e.g. screenreaders and search tools).
```
Comment (pkra) [inline sloan-draft.tex:28]: I felt that the sentence was off. There was no gap-to-be-bridged apparent so I re-wrote it a bit.

File: sloan-draft.tex [hunk 2]
```diff
@@ -15,52 +15,45 @@
...
-\item Develop a unified catalogue of public domain mathematics
-  materials, and enable the manual correction of recognised
-  mathematical formulae via crowdsourcing techniques, adapting a
-  crowdsourcing platform like Zooniverse's {\sf Scribe}.
-\item Building on techniques developed in the EuDML project,
-  investigate and implement methods for further enhancement of
-  content, e.g. via interlinking formulas and definitions, within and
-  between documents hosted on PlanetMath and PeerLibrary as well as to
-  outside sources like Arxiv, Wikipedia, Mathworld, etc., in support
-  of effective curation and online provision to readers.
```
Comment (pkra) [inline sloan-draft.tex:53]: I was mostly focused on shaving off lines here. 
- The first item seems off to me as it described the overall project, not a first step.
- The last item was hard to re-write. I wanted to get a similar rhythm into it but failed. I would drop the EuDML reference if you don't insist; it fits better in a full proposal.

File: sloan-draft.tex [hunk 3]
```diff
@@ -15,52 +15,45 @@
...
+content in the future global digital mathematics library.
 
-We envision that the resulting mathematical content will not only be
-directly useful for practitioners and learners, but also that the
-inclusion of historical material will be of interest to historians and
-other scholars and hobbyists, thereby nurturing a user community for
-future correction and enhancement activities.  As such we envision the
-proposed workflow to become a core component of a future global
-digital mathematics library with a emphasis on making high-quality
-mathematical content accessible by electronic means.
```
Comment (pkra) [inline sloan-draft.tex:63]: I re-wrote this quite a bit. Not sure I still capture the original intent.

**Comment by holtzermann17** [comment]
Looks good.  Merging.  I'll continue to adjust A-D a little.

**Pull request merged**